### DISCUSSION TITLE
Setting master password not working

### DISCUSSION THREAD

**Discussion by francois-egner** [dani-garcia/vaultwarden#3920] [Q&A]
### Subject of the issue
Whenever I try to login to my account I will get asked to setup a master password which I cannot remember setting up in the past. I did enter everything mandatory and receive an error that is not specified any further.

### Deployment environment

* vaultwarden version: 1.25.2

* Install method: Docker (:latest tag)

* Clients used: Mac Desktop App

* Reverse proxy and version: Nginx (The one that ships with the lattest nginx proxy manager)

* Other relevant details: 

### Steps to reproduce

1. Get latest Bitwarden Client for Mac
2. Use the :latest docker image available
3. Try to login to your account
4. Enter 2FA information/details
5. Fill out mandatory information for setting up a master password as asked

### Expected behaviour
I expect the master password to be set and being redirected into my vault

### Actual behaviour
The client says that an error occured. "The requestsest resource could not be found"

### Troubleshooting data
Log data:
vaultwarden    | [2023-09-24 19:12:39.663][_][ERROR] No matching routes for POST /api/accounts/set-password application/json.
vaultwarden    | [2023-09-24 19:12:39.663][_][WARN] No 404 catcher registered. Using Rocket default.
vaultwarden    | [2023-09-24 19:12:39.663][response][INFO] 404 Not Found

**Comment by stefan0xC** [comment]
Can you update to a newer Vaultwarden version? The issue should already be fixed by #3813 
Alternatively, use an older version of the desktop client.
